autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.css.d.ts

```diff
@@ -11,8 +11,12 @@ declare namespace CourseSelectColumnCssNamespace {
     courseSelectColumn: string;
     "expanded-row": string;
     "expanded-row-small": string;
+    "expanded-row-temp": string;
     expandedRow: string;
     expandedRowSmall: string;
+    expandedRowTemp: string;
```

Comment:
is `Temp` in `expandedRowTemp` intentional? Or was it like a testing name

Reply:
Intentional, because the class is only applied for a short time before it is replaced by either `expandedRow` or `expandedRowSmall`